vagga.yaml

```diff
@@ -100,12 +111,12 @@ commands:
       app: *run-app
       db: *run-db
 
-  test: !Command
-    description: Run tests
-    container: test
+  lint: !Command
+    description: Run linters
+    container: lint
     run: pre-commit run --all-files
 
-  pre-commit: !Command
-    description: Run pre-commit
+  test: !Command
+    description: Run tests
     container: test
-    run: [pre-commit]
+    run: python3 manage.py test --pattern="*_test.py"
```

Comment:
No, I haven't. I think, it'll work, but then I should rename test files to 'test_*.py' pattern.